projects/canopy/src/lib/spinner/docs/guide.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Spinner/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Spinner
+
+<p class="standfirst">A spinner lets the user know that something is happening. They are normally used on form submissions or loading pages when the data retrieval is typically slow.</p>
+
+![example](docs/spinner/example.png)
+
+## Usage
+
+This component should be used in instances where the duration of a process can't be determined.
+
+Import the component in your application:
+
+```
```

Comment:
Could you add `js` to the code snippet please?

Reply:
done 👍